includes/Plugins.php

```diff
@@ -9,6 +9,19 @@
  * List of Plugin Slugs/URLs/Domains
  */
 final class Plugins {
+
+	/**
+	 * Hiive flags to install and activate specific plugins
+	 *
+	 * @var array
+	 */
+	public static function get_hiive_plugin_flags() {
+		return array(
+			'hasEcomdash'     => true,
+			'hasYithExtended' => true,
+		);
+	}
```

Comment:
We can get rid of this in relation to my other comment on the onboarding PR.

Reply:
Sure